Sources/AWSSDKSwiftCore/HTTP/HTTPClient.swift

```diff
@@ -0,0 +1,37 @@
+//
```

Comment:
Does it make sense to rename this file to `AWSHTTPClient`?

Reply:
yep

Reply:
sorted